src/sdk/account/toNexusAccount.ts

```diff
@@ -192,6 +192,8 @@ export const toNexusAccount = async (
     }
   })
 
+  // Review:
```

Comment:
if we chose to do this upon every deployment. easiest thing is to pass attesters here as our botostrap supports it underneath
@joepegler

Reply:
Will attesters always be the same ?

Reply:
generally sholud be only one contract. but we have a freedom to pass right?

Reply:
can ask more questions to rhinestone guys

Reply:
feel free to open a thread!